packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -174,6 +174,37 @@ export class Loan {
   async createVault (vault: CreateVault, utxos: UTXO[] = []): Promise<string> {
     return await this.client.call('createvault', [vault.ownerAddress, vault.loanSchemeId, utxos], 'number')
   }
+
+  /**
+   * Deposit to vault
+   *
+   * @param {DepositVault} vault
+   * @param {string} vault.id Vault id
+   * @param {string} vault.from Collateral address
+   * @param {string} vault.amount In "amount@symbol" format
+   * @param {UTXO[]} [utxos = []] Specific UTXOs to spend
+   * @param {string} utxos.txid Transaction Id
+   * @param {number} utxos.vout Output number
+   * @return {Promise<string>}
+   */
+  async depositToVault (vault: DepositVault, utxos: UTXO[] = []): Promise<string> {
+    return await this.client.call('deposittovault', [vault.id, vault.from, vault.amount, utxos], 'number')
+  }
+
+  /**
+   * Take loan
+   *
+   * @param {TakeLoanMetadata} metadata
+   * @param {string} vaultId Vault id
+   * @param {string} amounts In "amount@symbol" format
```

Comment:
```suggestion
   * @param {string} loan.amounts In "amount@symbol" format
```

Reply:
you should multi-select lines the proposed suggestion next time
resolve @see https://github.com/DeFiCh/jellyfish/pull/661#discussion_r712259617

Reply:
Okay, I am not good at this. Haha